### ISSUE TITLE
Allow peeking at definition

### ISSUE THREAD

**Issue by sqs** [sourcegraph/sourcegraph.com#383]
Right now, there's no way to "see" or "peek" at the definition of something. You need to jump to it, which is a heavy, stateful operation. It would be nice if you could quickly peek at a definition.

> https://sourcegraph.com/github.com/golang/go@ac452349e4f40c30d50f9922bf2c4592e748ce5e/.tree/src/net/http/server.go#selected=2636

> Here, on the right I see usages, and I have "Go to definition" button. Instead, I would love to be able to have also "See definition" in order not to be forced to leave the page.

HT @slawosz

**Comment by gbbr** [comment]
One way to do this is to right click on the token and select "Open link in
New Tab" (or Window). That way you will keep your current page open. Is
that a reasonable use case for your need? @slawosz

**Comment by shurcooL** [comment]
It's not currently possible to right click the "Go to definition" button, which is what the original description mentioned. @gbbr, is there another way to achieve the equivalent effect of pressing that button, one where it is possible to right click and open in new tab?

**Comment by gbbr** [comment]
Yeah that should be an easy fix

**Comment by slawosz** [comment]
I think it would be easier to track code if you could see definition in current usages window.

**Comment by gbbr** [comment]
I agree that it would be highly beneficial to more easily open definitions in a new window, as suggested by @shurcooL, but I am not convinced that the current usage examples window should contain snippets of code. It seems a bit like it would clutter too much information. Opening a new window or tab should be easy enough once we get this through.

**Comment by gbbr** [comment]
I am more than happy and open to being convinced otherwise. I'm not trying to discourage you and feedback is always appreciated :)
